src/main/java/io/lettuce/core/AbstractRedisClient.java

```diff
@@ -297,6 +301,17 @@ protected void channelType(ConnectionBuilder connectionBuilder, ConnectionPoint
         }
     }
 
+    protected void resolver(ConnectionBuilder connectionBuilder, ConnectionPoint connectionPoint) {
+
+        LettuceAssert.notNull(connectionPoint, "ConnectionPoint must not be null");
+
+        if (connectionPoint.getSocket() == null) {
```

Comment:
[Supplemental Comments]
If `connectPoint.getSocket()` is not null then it means we try to use unix domain socket.
In this case we do not need to DNS resolver, I think.